Day-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Card.css';
+
+function CardBanner(props){
+    return(
+        <img className="card__img" src = {props.src} alt="react logo"/>
```

Comment:
Maybe get rid of the spaces around the src property to better match the surround code formatting.